From definition to principles. (#89)
ContributorCovenant/ethicalsource: content/definition.md

> +++

Reviewer: Shouldn't this be a redirect instead of a copy of /principles ?

title = "The Ethical Source Principles"
template = "definition"
heading="The Ethical Source Principles"
+++

The practice of Ethical Source is informed by these principles:

> 1. **Our work benefits the commons.** We release our software under conditions that do not prohibit modification, derivative works, and similar use and reuse in the general spirit of free, open source, source available, or other commons distribution.

Reviewer: Do we want to limit this definition to software or open it up more broadly? "contributions," "digital commons," "work," all come to mind.

> 1. **Our work benefits the commons.** We release our software under conditions that do not prohibit modification, derivative works, and similar use and reuse in the general spirit of free, open source, source available, or other commons distribution.

Reviewer: ```suggestion
1. **Our work benefits the commons.** We release our software under conditions that allow modification, derivative works, and similar use and reuse in the general spirit of free, open source, source available, or other commons distribution.
```

Or "promote."

Reviewer: Do we need to note that it's ok to limit the reuse within a given community or something of that nature? Not super sure how to phrase that.

2. **Our work is done in the open.** We make our source code publicly available, develop and maintain it in public view, and welcome  public contributions.

> 3. **We strive to make our community welcoming and just.** We have clear and documented rules for governance, and we consistently and fairly enforce a comprehensive code of conduct.

Reviewer: I'd be inclined to add something here about favoring autonomy of contributors, consensus decision-making, etc. But might be too specific for now.

4. **Our work puts accessibility first.** We design our software to be accessible, ensuring that all features and functionality are available to all users, including users who may rely on assistive devices.

> 5. **Our work prioritizes user safety.** We design our software with features and safeguards to minimize the risk of abuse or harm to others through the its use or misuse.

Reviewer: s/users/people/?

```suggestion
5. **Our work prioritizes the safety of people.** We design our software with features and safeguards to minimize the risk of abuse or harm to others through the its use or misuse.
```

> 6. **Our work protects user privacy.** We design our software with controls for the complete deletion of any personal data gathered from its use, and for exporting any such data to the end-user on request in a human-readable or interoperable and open format.

Reviewer: s/users/people/?

Also requesting both human readable and interoperable format, not one or the other.

```suggestion
6. **Our work protects privacy.** We design our software with controls for the complete deletion of any personal data gathered from its use, and for exporting any such data to people requesting it in both a human-readable format and in an interoperable and open format.
```

> 7. **Our work deserves to be supported.** We are empowered to request remuneration in the form of code contributions, financial contributions, or other forms of voluntary support from organizations that derive commercial value from our software.

Reviewer: I'm uneasy with this framing which implies power dynamics in the relationship that favor users of the commons in opposition to those giving away their work.

Not sure how to frame it, but maybe taking a step back and expecting that everyone deserves dignity and proper pay, or something like that?

> We measure our success in adhering to these principles on _impact_ rather than _intent_. 

Reviewer: 🔥🔥🔥

![https://img.shields.io/badge/ethical-user%20safety-%23bb8c3c?labelColor=393162](https://img.shields.io/badge/ethical-user%20safety-%23bb8c3c?labelColor=393162) [https://img.shields.io/badge/ethical-user%20safety-%23bb8c3c?labelColor=393162](https://img.shields.io/badge/ethical-user%20safety-%23bb8c3c?labelColor=393162)

![https://img.shields.io/badge/ethical-privacy-%23bb8c3c?labelColor=393162](https://img.shields.io/badge/ethical-privacy-%23bb8c3c?labelColor=393162) [https://img.shields.io/badge/ethical-privacy-%23bb8c3c?labelColor=393162](https://img.shields.io/badge/ethical-privacy-%23bb8c3c?labelColor=393162)

![https://img.shields.io/badge/ethical-compensation-%23bb8c3c?labelColor=393162](https://img.shields.io/badge/ethical-compensation-%23bb8c3c?labelColor=393162) [https://img.shields.io/badge/ethical-compensation-%23bb8c3c?labelColor=393162](https://img.shields.io/badge/ethical-compensation-%23bb8c3c?labelColor=393162)

## Attribution and Contributions

> Maintaining the Ethical Source Principles is the collaborative work of the [Ethical Source Working Group](/apply). Pull requests and issues in our [source code repository](https://github.com/ethicalSource/ethicalsource.dev) are encouraged. Special thanks are extended to [Nate Berkopec](https://nateberkopec.com), [Will Barton](https://github.com/willbarton/), [Steve Phillips](https://tryingtobeawesome.com), and [Madalyn Parker](https://twitter.com/madalynrose) for their contributions.

Reviewer: ```suggestion
Maintaining the Ethical Source Principles is the collaborative work of the [Organization for Ethical Source](/apply). Pull requests and issues in our [source code repository](https://github.com/ethicalSource/ethicalsource.dev) are encouraged. Special thanks are extended to [Nate Berkopec](https://nateberkopec.com), [Will Barton](https://github.com/willbarton/), [Steve Phillips](https://tryingtobeawesome.com), and [Madalyn Parker](https://twitter.com/madalynrose) for their contributions.
```
